homework/homework111223.py

```diff
@@ -0,0 +1,51 @@
+"""HOMEWORK4."""
+import logging
+
+logging.basicConfig(
+    level=logging.INFO,
+    format='[%(asctime)s] %(levelname)s: %(message)s',
+)
+
+# Text from teacher
+poem_text = """
+I wandered lonely as a cloud
+That floats on high o'er vales and hills,
+When all at once I saw a crowd,
+A host, of golden daffodils;
+Beside the lake, beneath the trees,
+Fluttering and dancing in the breeze.
+
+Continuous as the stars that shine
+And twinkle on the Milky Way,
+They stretched in never-ending line
+Along the margin of a bay:
+Ten thousand saw I at a glance,
+Tossing their heads in sprightly dance.
+"""
+
+poem_text.lower()
```

Comment:
Here you have real bug - result of this operation should be saved 
```suggestion
poem_text = poem_text.lower()
```